fdbserver/include/fdbserver/RocksDBCheckpointUtils.actor.h

```diff
@@ -285,6 +296,10 @@ ICheckpointReader* newRocksDBCheckpointReader(const CheckpointMetaData& checkpoi
                                               const CheckpointAsKeyValues checkpointAsKeyValues,
                                               UID logID);
 
+IRocksDBSstFileWriter* beginRocksDBSstFileWriter(std::string localFile);
```

Comment:
nit: rename `newRocksDBSstFileWriter`, and return type as `std::unique_ptr<IRocksDBSstFileWriter>` to help you manage the life time of `RocksDBSstFileWriter`, then `endRocksDBSstFileWriter()` can be removed.